lib/Common/Core/Output.cpp

```diff
@@ -306,6 +308,7 @@ Output::PrintBuffer(const char16 * buf, size_t size)
         if (s_file == nullptr || Output::s_capture)
         {
             bool addToBuffer = true;
```

Comment:
Looks like this line can/should be inside the `#ifdef` as well, since it only applies when buffering.

Reply:
Fixed.